packages/shared/src/urbit/settings.ts

```diff
@@ -1,6 +1,8 @@
+import { ThemeName } from 'tamagui';
+
 import { DisplayMode, SortMode } from './channel';
 
-export type Theme = 'light' | 'dark' | 'auto';
+export type AppTheme = ThemeName | 'auto';
```

Comment:
Not stoked on the duplicate declaration but gets us around the circular dependency issue